pkg/kubectl/stop.go

```diff
@@ -203,7 +203,7 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string, timeout
 	}
 	if len(overlapRCs) > 0 {
 		return fmt.Errorf(
-			"Detected overlapping controllers for rc %v: %v, please manage deletion individually with --cascade=false.",
+			"overlapped controllers are detected for rc %v: %v, please manage deletion individually with --cascade=false.",
```

Comment:
`kubectl stop` is deprecated, so I'm not sure if it's worthy to send a patch for fixing anything of it...

Reply:
This file is actually used by `kubectl delete` instead of `kubectl stop`. The file has been renamed to avoid confusion like this.